perception/image_projection_based_fusion/docs/roi-detected-object-fusion.md

```diff
@@ -27,23 +30,26 @@ The `DetectedObject` has three shape and the polygon vertices of a object are as
+| Name                    | Type                                                  | Description                                       |
+| ----------------------- | ----------------------------------------------------- | ------------------------------------------------- |
+| `output`                | `autoware_auto_perception_msgs::msg::DetectedObjects` | detected objects                                  |
+| `output/image_rawID`    | `sensor_msgs::msg::Image`                             | images for visualization, `ID` is between 0 and 7 |
+| `debug/fused_objects`   | `autoware_auto_perception_msgs::msg::DetectedObjects` | fused detected objects                            |
+| `debug/ignored_objects` | `autoware_auto_perception_msgs::msg::DetectedObjects` | not fused detected objects                        |
 
 ## Parameters
 
 ### Core Parameters
 
-| Name            | Type  | Description                                                                    |
-| --------------- | ----- | ------------------------------------------------------------------------------ |
-| `use_iou_x`     | bool  | calculate IoU only along x-axis                                                |
-| `use_iou_y`     | bool  | calculate IoU only along y-axis                                                |
-| `use_iou`       | bool  | calculate IoU both along x-axis and y-axis                                     |
-| `iou_threshold` | float | the IoU threshold to overwrite a label of a detected object with that of a roi |
-| `rois_number`   | int   | the number of input rois                                                       |
-| `debug_mode`    | bool  | If `true`, subscribe and publish images for visualization.                     |
+| Name                                            | Type   | Description                                                                                               |
+| ----------------------------------------------- | ------ | --------------------------------------------------------------------------------------------------------- |
+| `rois_number`                                   | int    | the number of input rois                                                                                  |
+| `debug_mode`                                    | bool   | If `true`, subscribe and publish images for visualization.                                                |
+| `use_iou`                                       | bool   | calculate IoU both along x-axis and y-axis                                                                |
+| `iou_threshold`                                 | double | the IoU threshold to                                                                                      |
+| `passthrough_lower_bound_probability_threshold` | double | If `existence_probability` of detected objects is greater than the threshold, they passthrough as outputs |
```

Comment:
how about `max target existence probability`?

Reply:
I want to clearfy that do nothing to detected obejcts if the exisitence probability of them is greater than the threshold.

Reply:
ok :+1: